tendrl/monitoring_integration/flows/delete_resource_from_graphite/__init__.py

```diff
@@ -0,0 +1,215 @@
+import etcd
+import os
+import datetime
+import ConfigParser
+
+
+from tendrl.commons import flows
+from tendrl.commons.utils import log_utils as logger
+from tendrl.commons.utils import etcd_utils
+from tendrl.monitoring_integration.grafana import create_dashboards
+
+
+
+class DeleteResourceFromGraphite(flows.BaseFlow):
+
+    def run(self):
+        super(UpdateGraphite, self).run()
+        cluster_id = self.parameters.get("TendrlContext.integration_id")
```

Comment:
this is not cluster_id, stop adding confusing code and use integration_id everywhere

Reply:
@r0h4n Made the required changes, Will make changes in other files also in separate PR